### ISSUE TITLE
[Compatibility Request]: Industrial Revolution by Redstone Rebooted

### ISSUE THREAD

**Issue by quentin452** [FalsePattern/EndlessIDs#104]
### Other mod's name

jp-plusplus-ir2

### Other mod's homepage

https://www.curseforge.com/minecraft/mc-mods/industrial-revolution-by-redstone-rebooted/screenshots

### Other mod's license

Other open source license (please specify in the description)

### Game log

[fml-client-latest.log](https://github.com/FalsePattern/EndlessIDs/files/10927600/fml-client-latest.log)

### Description

mod dependency : mceconomy2

```
licence : 

Version 1.0.4

March 6, 2016

0.Basic
All things that have not been specified in this license are treated as those that ban in principle.
All the contents of this license, the owner of this project (plusplus_Farore, plusplus_F) is assumed to be arbitrarily changed,
mod that this license is applied, always it is assumed that the license of the latest version has been applied.

1.Scope
All mod that this license is applied, if you follow and agree to this license only,
mod jar files, source code, binary, and any or all of the resources,
Incorporate into their ownership to Minecraft, to see in their own development and within the mod, it is possible to browse by means other than Minecraft.

jar file of the mod that this license is applied, source code, binary, and resources, both can not be other than the author makes a modification.

Commercial use of the mod that this License applies to the ban.
The commercial here, refers to sales, affiliate, that directly or indirectly financial benefit in such creators incentive program is generated.

Secondary distribution of mod(Ex. jar files, source code, distribution of binary and all of the resources) that this license is applied, as a rule without the express written consent of the commercial and non-commercial.
However, if it is permitted directly from the author only for non-commercial purposes, it is assumed to be able to perform the secondary distribution of mod in the form specified by the author.
In addition, such as video or add-ons, for this license is applied is a result of using the mod obtained artifacts, you can be free to the public only for non-commercial.

However, for the binary obtained by to build the source code of the mod that this license is applied, and ban the public.

For config files that are generated by the mod that this license is applied, it is possible to freely edit and publish only to non-commercial.

2.Other
The user of the mod which the license is applied, for the license, it is possible to perform a query to the owner of the topics intended to clarify the treatment.
```

crash when make a world

**Comment by FalsePattern** [comment]
License is kinda weird, but it doesn't seem to forbid patching

**Comment by FalsePattern** [comment]
Fix scheduled for 1.5 alpha 7

**Comment by quentin452** [comment]
fixed
